packages/server/src/models/objection/UserNotification.js

```diff
@@ -25,4 +26,10 @@ export default class UserNotification extends BaseModel {
       },
     }
   }
+
+  static findUnsent() {
+    return this.query()
+      .whereNull('sentAt')
+      .andWhere('createdAt', '>', new Date(Date.now()-HOURS))
```

Comment:
Haha, we can do this... but let's use something like [dayjs](https://github.com/iamkun/dayjs) (or [date-fns](https://date-fns.org/), but we primarily use dayjs on the frontend because it's much, much lighter).

It's not a huge problem in the backend because we don't have to be concerned about different operating environments (e.g. client timezones), but usually still better to use a tried and true library for these kinds of manipulations.